project/password/templates/registration/password_change_complete.html

```diff
@@ -0,0 +1,8 @@
+{%extends 'base.html' %}
+
+{%block content%}
+<h2>Your password has been changed</h2>
```

Comment:
Add internationalization (i18n)
```suggestion
<h2>
    {% translate "Your password has been changed" %}
</h2>
```